rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/scan/action.py

```diff
@@ -0,0 +1,71 @@
+import ast
+
+import insightconnect_plugin_runtime
+from .schema import ScanInput, ScanOutput, Input, Component
+# Custom imports below
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+class Scan(insightconnect_plugin_runtime.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name='scan',
+            description=Component.DESCRIPTION,
+            input=ScanInput(),
+            output=ScanOutput())
+
+    def asset_search(self, hostname: str, ip: str):
```

Comment:
Is this asset_search ever something we would potentially use elsewhere?

Reply:
Not this one. There is a more universal function to format response in asset_search